src/main/java/net/minestom/server/entity/pathfinding/PFBlockDescription.java

```diff
@@ -76,74 +83,42 @@ public boolean isFullyBounded() {
             return false;
         }
         // All doors/trapdoors.
-        if (block.name().toUpperCase().endsWith("DOOR")) {
+        if (block.getName().toUpperCase().endsWith("DOOR")) {
             return false;
         }
-        if (block.name().toUpperCase().startsWith("POTTED")) {
+        if (block.getName().toUpperCase().startsWith("POTTED")) {
             return false;
         }
         // Skulls & Heads
-        if (block.name().toUpperCase().contains("SKULL") || block.name().toUpperCase().contains("HEAD")) {
+        if (block.getName().toUpperCase().contains("SKULL") || block.getName().toUpperCase().contains("HEAD")) {
             return false;
         }
         // Carpets
-        if (block.name().toUpperCase().endsWith("CARPET")) {
+        if (block.getName().toUpperCase().endsWith("CARPET")) {
             return false;
         }
         // Slabs
-        if (block.name().toUpperCase().contains("SLAB")) {
+        if (block.getName().toUpperCase().contains("SLAB")) {
             return false;
         }
         // Beds
-        if (block.name().toUpperCase().endsWith("BED")) {
+        if (block.getName().toUpperCase().endsWith("BED")) {
             return false;
         }
         // Glass Panes
-        if (block.name().toUpperCase().endsWith("PANE")) {
+        if (block.getName().toUpperCase().endsWith("PANE")) {
             return false;
         }
```

Comment:
Not really related to this PR but this should absolutely be replaced with something that doesn't rely on names at some point.

Reply:
This can be done using the Hitbox actually which would be better for the behaviour. But all the hitbox code is down to @TheMode. I don't even know if I am extracting the correct hitbox data.